### PR TITLE
Add ability to set the svg drawing without adding to the undo stack.

### PR DESCRIPTION
**Pull request by iuyiuy**
When loading a file on init, it's useful to set it without it being added to the undo stack.

### CODE REVIEW

File: editor/svgcanvas.js [hunk 1]
```diff
@@ -4596,7 +4596,7 @@ this.setSvgString = function(xmlString) {
 		svgedit.path.clearData();
 		svgroot.appendChild(selectorManager.selectorParentGroup);
 		
-		addCommandToHistory(batchCmd);
+		if(!preventUndo) addCommandToHistory(batchCmd);
```
Comment (codedread) [inline editor/svgcanvas.js]: nit: add a space between the if and the (

also, can you please correct the other two instances of that in this file?

Comment (iuyiuy) [reply editor/svgcanvas.js]: I only found one other instance of it in this branch, but i've changed that now.

I might have done it in that style in other commits / pull requests. Sorry about that. I didn't realize that was the coding style of this project. I'll try to do that in the future.

File: editor/svgcanvas.js [hunk 2]
```diff
@@ -4442,7 +4442,7 @@ var convertToGroup = this.convertToGroup = function(elem) {
 //
 // Returns:
 // This function returns false if the set was unsuccessful, true otherwise.
```
Comment (codedread) [inline editor/svgcanvas.js]: Please update the docs of this method to mention the optional preventUndo parameter

Comment (iuyiuy) [reply editor/svgcanvas.js]: I've added the documentation for it now.

**Pull request merged**